src/components/Filters/Filters.tsx

```diff
@@ -58,7 +58,7 @@ const getDefaultState = ([firstField]: Field[]) => {
   ];
 };
 
-const Form = styled.form`
+const Form = styled.div`
```

Comment:
Could we rename `Form` to something else if it is not from anymore? Also you can `FlexContainer` instead